app/code/Magento/SendFriend/view/frontend/templates/send.phtml

```diff
@@ -73,6 +74,9 @@
             <div class="control">
                 <input name="sender[email]" value="<?php echo $block->escapeHtml($block->getEmail()) ?>"
                        title="<?php /* @escapeNotVerified */ echo __('Email') ?>" id="sender-email" type="text" class="input-text"
+                <input name="sender[email]" value="<?= $block->escapeHtmlAttr($block->getEmail()) ?>"
```

Comment:
The method `escapeHtmlAttr` is absent in the `\Magento\SendFriend\Block\Send` in the Magento 2.1 version